.github/CODEOWNERS

```diff
@@ -220,3 +230,12 @@
 /pkg/util/stop               @cockroachdb/server-prs
 /pkg/util/tracing            @cockroachdb/obs-inf-prs
 /pkg/workload/               @cockroachdb/sql-experience
+
+# Own all bazel files to dev-inf.
+**/BUILD.bazel               @cockroachdb/dev-inf
+# Avoid mass pings when updating proto tooling.
+# For some reason, **/*.pb.go does not work (in the
+# sense that ./pkg/cmd/whoownsit will not match this
```

Comment:
@otan do you have any insight on that? Must be a bug in our code, no?

Reply:
For more context, `BUILD.bazel` as a rule doesn't work either - I *think* it was supposed to - that's why I started on `**/BUILD.bazel` in the first place.

Reply:
Probably the library we use in codeowners: "github.com/zabawaba99/go-gitignore"